toolkit/tools/internal/logger/log.go

```diff
@@ -86,7 +86,7 @@ func initLogFile(filePath string, color string) (err error) {
 		return
 	}
 
-	file, err := os.Create(filePath)
+	file, err := os.OpenFile(filePath, os.O_CREATE|os.O_WRONLY|os.O_APPEND, 0644)
```

Comment:
This function is used across all of our tools and we don't want to append to all logs by default - people often run multiple builds, which end up logging to the same files, so this would get messy very soon.

Reply:
Would adding a new param to the function to switch between append and truncate and let truncate be the default minimize the impact?

No truncation in a single invocation of the toolkit and starting a new log file from scratch in each toolkit invocation are the goals here.

Reply:
We definitely want fresh logs for each tool when it starts by default, I think append should be the special case.

Reply:
I agree that the append would be a special case. In general I'm fine with making append an option but in this particular scenario, I think the issue lies with us trying to combine makefile's output with logs coming from a single tool. To keep the tool isolated and simpler, I'd allow it to continue to only worry about printing its own logs into a file and leave it up to the caller, if they want to do something more fancy. For instance we could print the tool's output to different files for each iteration of the `for url in $(PACKAGE_URL_LIST); do` loop. Or we could print it to a temporary file log and append the main log with its contents for each loop iteration.